.ci/install_runtime.sh

```diff
@@ -54,7 +54,7 @@ fi
 
 enable_hypervisor_config(){
 	local path=$1
-	sudo mv "$path" "${PKGDEFAULTSDIR}/configuration.toml"
+	sudo ln -f "$path" "${PKGDEFAULTSDIR}/configuration.toml"
```

Comment:
You mentioned in the commit copy, but seeing a link here. Do you want that? Any changes to one, will be reflected on the other.

Reply:
@amshinde thank you, I updated the commit to match the change